submissions/Roka20012/pop-up-menu/index.html

```diff
@@ -8,32 +8,33 @@
   </head>
   <body>
     <div class="container">
+      <p>MENU HERE! --------></p>
       <label class="check-container" for="chekcme"><span class="checkmark"></span></label>
       <input class="check" id="chekcme" type="checkbox" name="chekcme">
       <input id="check-show-more" type="checkbox">
       <div class="mark"></div>
       <div class="container-menu">
         <div class="sprite-container"><a href="">
-            <div class="sprite1"></div><span>Phone</span></a><a href="">
-            <div class="sprite2"></div><span>Guard</span></a><a href="">
-            <div class="sprite3"></div><span>Document</span></a><a href="">
-            <div class="sprite4"></div><span>Maps</span></a><a href="">
-            <div class="sprite5"></div><span>Excel</span></a><a href="">
-            <div class="sprite6"></div><span>Mark</span></a><a href="">
-            <div class="sprite7"></div><span>I`m off</span></a><a href="">
-            <div class="sprite8"></div><span>Locked</span></a><a href="">
-            <div class="sprite9"></div><span>No no..</span></a><a href="">
-            <div class="sprite10"></div><span>School?</span></a><a class="hide" href="">
-            <div class="sprite7"></div><span>I`m off</span></a><a class="hide" href="">
-            <div class="sprite8"></div><span>Locked</span></a><a class="hide" href="">
-            <div class="sprite9"></div><span>No no..</span></a><a class="hide" href="">
-            <div class="sprite10"></div><span>School?</span></a><a class="hide" href="">
-            <div class="sprite7"></div><span>I`m off</span></a><a class="hide" href="">
-            <div class="sprite8"></div><span>Locked</span></a><a class="hide" href="">
-            <div class="sprite9"></div><span>No no..</span></a><a class="hide" href="">
-            <div class="sprite10"></div><span>School?</span></a><a class="hide" href="">
-            <div class="sprite7"></div><span>I`m off</span></a><a class="hide" href="">
-            <div class="sprite8"></div><span>Locked</span></a></div>
+            <div class="sprite"></div><span>Phone</span></a><a href="">
```

Comment:
Check your `<a>` tag

Reply:
If you mean add `href="#"` i fix it if i am wrong please correct me)

Reply:
No, I mean your link starting from closing tag `</a>` and only then opening tag `<a class="hide" href="#">` but it must be the other way around.